doc/jekyll_tools/img/igraph2.svg

```diff
@@ -0,0 +1,185 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I'd probably clean up the `img` folder as well and remove all the images that are not strictly needed for testing the documentation.

Reply:
done, I left both igraph logos thinking we might need it if we prettify it